### PR TITLE
Release 1.6.0

### PR DESCRIPTION
**Pull request by jgsqware**
Signed-off-by: Julien Garcia Gonzalez <julien@giantswarm.io>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -63,4 +65,4 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 - Tolerations changed to tolerate all taints.
 - Change prioty class to `giantswarm-critical`.
 
-[Unreleased]: https://github.com/giantswarm/net-exporter/compare/v1.2.0...HEAD
+[unreleased]: https://github.com/giantswarm/net-exporter/compare/v1.2.0...HEAD
```
Comment (kopiczko) [inline CHANGELOG.md]:
```suggestion
[Unreleased]: https://github.com/giantswarm/net-exporter/compare/v1.2.0...HEAD
```

**Comment by tomahawk28** [comment]
While I'm using `1.6.0` on 9.1.1 WIP cluster, I got below error. 
```
D 02/07 12:05:42 /apis/application.giantswarm.io/v1alpha1/namespaces/giantswarm/charts/net-exporter releasev1.ApplyCreateChange helm release `net-exporter` failed | chart-operator/service/controller/chart/v1/resource/release/create.go:58 | event=update | version=2314
	/go/src/github.com/giantswarm/chart-operator/vendor/github.com/giantswarm/helmclient/helmclient.go:334:
	/go/src/github.com/giantswarm/chart-operator/vendor/github.com/giantswarm/helmclient/helmclient.go:381:
	/go/src/github.com/giantswarm/chart-operator/vendor/github.com/giantswarm/backoff/retry.go:23:
	/go/src/github.com/giantswarm/chart-operator/vendor/github.com/giantswarm/helmclient/helmclient.go:371:
	rpc error: code = Unknown desc = render error in "net-exporter/templates/daemonset.yaml": template: net-exporter/templates/daemonset.yaml:52:22: executing "net-exporter/templates/daemonset.yaml" at <.Values.NetExporter.DNSCheck.TCP.Disabled>: nil pointer evaluating interface {}.DNSCheck
```

Where `NetExporter` supposed to come from?

**Comment by kopiczko** [comment]
Can you run `opsctl update draughtsman -i INSTALLATION` and try again?

**Comment by tomahawk28** [comment]
Actually, it was about appcatalog configmap, which was not updated as our installation repo. 
So I did, 
```  opsctl ensure appcatalogs --installation gorgoth --user jhbok --no-cache```

**Pull request merged**